cmd/addons.go

```diff
@@ -116,4 +117,32 @@ var (
 			autocomplete.AddonsUpgradeAutoComplete(c)
 		},
 	}
+	AddonsInfoCommand = cli.Command{
+		Name:     "addons-info",
+		Category: "Addons",
+		Usage:    "Display infos about add-ons attached to your app",
+		Flags:    []cli.Flag{appFlag, addonFlag},
+		Description: ` Display infos about add-ons attached to your app:
+    $ scalingo --app my-app addons-info --addon <addon-id>
```

Comment:
Uncountable + use the full word
```suggestion
		Usage:    "Display information about an add-on attached to your app",
		Flags:    []cli.Flag{appFlag, addonFlag},
		Description: ` Display information about an add-on attached to your app:
    $ scalingo --app my-app addons-info --addon <addon-id>
```